app/webpacker/css/admin_v3/components/pagination.scss

```
@@ -39,10 +39,16 @@
       cursor: pointer;
     }
 
-    &:hover:not(.disabled) {
+    &:hover:not(.disabled):not(.gap) {
```

Building upon my previous comment, how about we just scope it to clickable links? 
```suggestion
    &[href] {
```

---

Yes! ad3f9001c3